packages/composite-checkout/src/components/grid-row.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Row = styled.div`
+	display: grid;
+	width: 100%;
+	grid-template-columns: ${props => props.columnWidths};
+	grid-column-gap: ${props => props.gap};
+	justify-items: stretch;
+`;
+
+export default function GridRow( { gap, columnWidths, className, children } ) {
+	return (
+		<Row gap={ gap } columnWidths={ columnWidths } className={ className }>
+			{ children }
+		</Row>
+	);
+}
+
+GridRow.propTypes = {
+	gap: PropTypes.string,
```

Comment:
`gap` is used in the `styled.div` call without a fallback, so we should either make this a required prop or provide a default value in either `GridRow` or the styled template string. The same is true of `columnWidths`.

Reply:
good point. I'll make them required.